components/server/src/server_utilities/functions.py

```diff
@@ -68,3 +74,34 @@ def percentage(numerator: int, denominator: int, direction: Direction) -> int:
     if denominator == 0:
         return 0 if direction == "<" else 100
     return int((100 * Decimal(numerator) / Decimal(denominator)).to_integral_value(ROUND_HALF_UP))
+
+
+def symmetric_encrypt(message: bytes) -> Tuple[bytes, bytes]:
+    """
+    Encrypt the given value using Fernet 32 byte key.
+    Returns the generated key and the encrypted message.
+    """
```

Comment:
DeepSource will complain about this docstring format I think.

Reply:
It seems to be ok with it for now. That being said, I think there could be a cleaner style to specify the returned value.